src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        int times = 3;
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < times; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
can we write just `System.out.println(lottery.getRandomBall());`?

Reply:
Of course!